portstyle.css

```diff
@@ -0,0 +1,229 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	padding: 0;
+	margin: 0;
+	color: #FFFFFF;
+}
+
+.fontStyleHead {
+	font-family: Arial, sans-serif;
+	font-size: 1.2rem;
+	color: #FFFFFF;
+}
+
+.navigationBar {
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	color: #FFFFFF;
+	background: #22303a;
+	position: fixed;
+	top: 0;
+	width: 100%;
+	z-index: 10;
+	border-bottom: 2px solid #c67e59;
+}
+
+.navigationBar a {
+	margin: 20px auto;
+	font-size: 1.5rem;
+	background: none;
+	border: none;
+	text-decoration: none;
+}
+
+
+.mainImage {
+	display: flex;
+	justify-content: center;
+	width: 100%;
+}
+
+
+.backgroundImage {
+	background-image: url("https://images.unsplash.com/photo-1592750088063-96b411cd5895?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80");
```

Comment:
again, download this image, dont link to the online version

Reply:
Done for all images.